specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json

```diff
@@ -0,0 +1,601 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-07-22-preview",
+    "title": "Azure Alerts Management Service Resource Provider",
+    "description": "Azure Alerts Management Service provides a single pane of glass of alerts across Azure Monitor."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/prometheusRuleGroups": {
```

Comment:
>prometheusRuleGroups [](http://example.com/codeflow?start=74&length=20)

Prometheus sounds like a project name. Can you use a more descriptive name for this resource? I don't think the name Prometheus should appear anywhere in this API definition, since (AFAICT) nobody outside of your RP would know what it refers to.

Reply:
It is alignment with the OSS naming, which is well known in to potential customers.